regolith/builders/preslistbuilder.py

```diff
@@ -99,6 +99,12 @@ def group_member_ids(self, grp):
 
     def latex(self):
         """Render latex template"""
+        # just a reminder placeholder how to access these.  These
+        # print statements will be removed when the builder is updated
+        # to use them!
+        print(self.rc.from_date)
+        print(self.rc.to_date)
```

Comment:
I am not sure this is useful to have

Reply:
@scopatz you are right, this is not useful.  I was using it for testing (making sure the values propageted through to the builder) and then I wanted to keep it there so I remembered the syntax, but there are other better ways of remembering the syntax so I will remove them.